pre-index.php

```diff
@@ -131,12 +163,82 @@
                             <span class="switch-handle"></span>
                         </label>
                     </div>
-                </div>';
-                }
+                </div>'; ?>
+                <div id="pokemon-filter-wrapper" style="display:none">
+                    <div id="tabs">
+                        <ul>
+                            <li><a href="#tabs-1">Hide Pokemon</a></li>
+                            <li><a href="#tabs-2">Excl. Min IV/Lvl</a></li>
+                        </ul>
+                        <div id="tabs-1">
+                            <?php
+                            if (!$noHidePokemon) {
+                                ?>
+                                <div class="form-control hide-select-2">
+                                    <label for="exclude-pokemon">
+                                        <div class="pokemon-container">
+                                            <input id="exclude-pokemon" type="text" readonly="true">
+                                            <?php
+                                            pokemonFilterImages($noPokemonNumbers); ?>
+                                        </div>
+                                        <a href="#" class="select-all">All</a><a href="#" class="hide-all">None</a>
+                                    </label>
+                                </div>
+                                <?php
+                            } ?>
+                        </div>
+                        <div id="tabs-2">
+                            <?php
+                            if (!$noExcludeMinIV) {
```

Comment:
We should probably remove the tabs completely instead of only removing their content (for `noHidePokemon` and `noExcludeMinIV`